Taskfile.yml

```diff
@@ -0,0 +1,33 @@
+version: '3'
+
+dotenv: ['.env']
+
+tasks:
+  run:
+    cmds:
+      - go run main.go
+  dev_server:
+    cmds:
+      - >
+        go run github.com/air-verse/air@v1.61.1 \
```

Comment:
The 'dev_server' command references a specific version of the air package ('v1.61.1'). Consider using a version restriction (e.g., ^v1.61.1) if updates are expected, or document the reasoning for this fixed version to avoid unintended broken builds in the future.